railties/lib/rails/generators/named_base.rb

```diff
@@ -26,6 +26,10 @@ def template(source, *args, &block)
             super
           end
         end
+
+        def js_template(source, destination)
```

Comment:
@kaspth No. By adding it in named base, it will be available for everyone which hooks into railties.

If we add it to `Js::Generators::ChannelGenerator` then coffee-rails has to hook into Action cable.

Also I am trying to remove `Js::Generators::ChannelGenerator` based on Mathhew's suggestions.